### ISSUE TITLE
Usage of format string literal necessitates Python 3.6

### ISSUE THREAD

**Issue by evictor** [chromano/wi#2]
https://github.com/chromano/wi/blob/ee5ddf40c62a28524fa205143cb97d5e94792610/wi#L102

Since the usage is so trivial, it would be nice if a more backwards-compatible approach were used instead.

**Comment by jayvdb** [comment]
Or .. https://github.com/asottile/future-fstrings ;-)

But I agree with removing the fstrings for projects/situations like this where simplicity and backwards compatibility are beneficial to users.

**Comment by evictor** [comment]
Much appreciated. Also did not know about future-fstrings, very useful 🙏

**Comment by carloshatch** [comment]
@evictor I totally agree with you. Also, I think it makes sense to go a little further and wrap any `print`s with a `log` or `output` function.
